Makefile

```diff
@@ -102,14 +91,7 @@ lint:
 	@find pkg cmd -name '*.go' | grep -v 'generated' | xargs -L 1 golint
 
 .PHONY: clean
-clean: container-clean bin-clean
-
-.PHONY: container-clean
-container-clean:
-	rm -rf .container-* .push-* dist
-
-.PHONY: bin-clean
-bin-clean:
+clean:
 	rm -rf .go bin
```

Comment:
No need to rm `.go` as it's not created any more AFAIK

Reply:
Still created on my machine. Will remove after fixing the build process in a subsequent PR.